### PR TITLE
Add AdminData model, endpoints and behaviour

### LINKED ISSUE #220: Add AdminData model and endpoints
Create `AdminData` model and related endpoints for `GET` and `UPDATE`

### PR DESCRIPTION
**Pull request by skovmand**
Fixes #220

### CODE REVIEW

File: lib/wise_homex/models/admin_data.ex [hunk 1]
```diff
@@ -0,0 +1,11 @@
+defmodule WiseHomex.AdminData do
+  @moduledoc false
+
+  use WiseHomex.BaseModel
+
+  embedded_schema do
+    belongs_to :account, WiseHomex.Account
```
Comment (lasseebert) [inline lib/wise_homex/models/admin_data.ex:7]: You would probably also need the opposite relation, right?

Comment (skovmand) [reply lib/wise_homex/models/admin_data.ex:7]: Ah yeah. I will add that!

Comment (skovmand) [reply lib/wise_homex/models/admin_data.ex:7]: Fixed in https://github.com/wise-home/wise_homex/pull/221/commits/a7b590ad90b78d872808172528b23c3117a12be4

**Comment by skovmand** [comment]
Reviewed and approved IRL

**Pull request merged**